### PR TITLE
Filter bugs per product

### LINKED ISSUE #5: Product drop down
Display a dropdown up right for bugs filtering. Populated by Products returned from bug list on event selection.

### PR DESCRIPTION
**Pull request by comzeradd**
- Fixes #5

### CODE REVIEW

**Comment by comzeradd** [comment]
@ppapadeas have a look

shot1

**Comment by ppapadeas** [comment]
r+ for UI

@johngian ?

File: reclama/templates/bugs.html [hunk 1]
```diff
@@ -22,7 +22,23 @@
...
+  </div>
+
+  <div class="container">
+    <div class="event-text">{{ event.text|safe }}</div>
+    <div class="products">
+      <form action="" method="post">
+        <div class="select">
+          <select id="product-selection" name="product">
+            <option selected value="0">Everything</option>
+
```
Comment (johngian) [inline reclama/templates/bugs.html]: Please remove empty line.

**Comment by johngian** [comment]
Code looks good.

**Comment by johngian** [comment]
You could use a lightweight databinding framework like `rivetsjs` [1] but if it works feel free to merge.

[1] http://rivetsjs.com/

**Comment by comzeradd** [comment]
Thanks for the review. Using rivetsjs would probably make sense to use it in the future if we want to add more similar functionality at the frontend. I'll keep this in mind.

**Pull request merged**